packages/benchmark/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "n8n-benchmark",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index",
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json && tsc-alias -p tsconfig.build.json",
+    "start": "./bin/n8n-benchmark",
```

Comment:
Are we adding prettier and the .vscode folder to make sure prettier is used to format and to format on save by default?

Reply:
Not sure I follow. Do we have some prettier config in `.vscode`? We have the `.prettierrc.js` in the repo root, which AFAIK should apply to all packages?

Reply:
I meant the formatter configuration in vscode as in here -> https://github.com/n8n-io/ai-assistant-service/blob/main/.vscode/settings.json

Reply:
Think we have those [already](https://github.com/n8n-io/n8n/blob/master/.vscode/settings.default.json#L2-L3)